aioesphomeapi/model.py

```diff
@@ -270,6 +289,15 @@ class ClimateInfo(EntityInfo):
     supported_swing_modes = attr.ib(
         type=List[ClimateSwingMode], converter=_convert_climate_swing_modes, factory=list
     )
+    supported_custom_fan_modes = attr.ib(
+        type=List[str], converter=_convert_string, factory=list
```

Comment:
```suggestion
        type=List[str], converter=list, factory=list
```

Just as an FYI - the `_convert_string` is equivalent to `list` converter.